site/docs/pages/getting-started.mdx

```diff
@@ -30,6 +30,48 @@ bun add @coinbase/onchainkit
 
 :::
 
+### API Key
+
+#### Obtain an API key from the [Coinbase Developer Platform APIs](https://portal.cdp.coinbase.com/products/onchainkit).
+
+<img
+  alt="OnchainKit copy API KEY"
+  src="https://onchainkit.xyz/assets/copy-api-key-guide.png"
+  width="auto"
+/>
+
+#### Create a .env file
+
+Create a new file in your project’s root directory and name it “.env”.
+
+<img
+  alt="OnchainKit copy API KEY"
+  src="https://onchainkit.xyz/assets/getting-started-create-env-file.png"
+  width="auto"
```

Comment:
the width is too large, also it feels very odd to see it like that.

Reply:
Updated! - See image in PR description